Weather/AppDelegate.swift

```diff
@@ -21,7 +21,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             return false
         }
         
-        // Maybe we could implement it as some kind of router?:
+        // ???: Could this be implemented as some kind of router?
```

Comment:
Interesting question, I am not too sure, and not sure what would be the value here. 
If you look at it, we ask the builder of the module we want as first module to build itself, we get the VC and display it.

Seems clear enough to me ... but open to new ideas :)